spec/traits.dd

```diff
@@ -1084,6 +1084,13 @@ static assert(__traits(getParameterStorageClasses, foo, 0)[1] == "ref");
 static assert(__traits(getParameterStorageClasses, foo, 1)[0] == "scope");
 static assert(__traits(getParameterStorageClasses, foo, 2)[0] == "out");
 static assert(__traits(getParameterStorageClasses, typeof(&foo), 3)[0] == "lazy");
+
+int* p, a;
+int b, c;
+
+static assert(__traits(getParameterStorageClasses, foo(p, a, b, c), 1)[0] == "scope");
+static assert(__traits(getParameterStorageClasses, foo(p, a, b, c), 2)[0] == "out");
+static assert(__traits(getParameterStorageClasses, foo(p, a, b, c), 3)[0] == "lazy");
```

Comment:
```
__stdin.d(15): Error: first argument to `__traits(getParameterStorageClasses, foo(p, a, b, delegate int() pure nothrow @nogc @safe => c), 1)` is not a function
__stdin.d(15):        while evaluating: `static assert((__error)[0] == "scope")`
__stdin.d(16): Error: first argument to `__traits(getParameterStorageClasses, foo(p, a, b, delegate int() pure nothrow @nogc @safe => c), 2)` is not a function
__stdin.d(16):        while evaluating: `static assert((__error)[0] == "out")`
__stdin.d(17): Error: first argument to `__traits(getParameterStorageClasses, foo(p, a, b, delegate int() pure nothrow @nogc @safe => c), 3)` is not a function
__stdin.d(17):        while evaluating: `static assert((__error)[0] == "lazy")`
```

I don't know which compiler version the documentation tester uses, but it doesn't seem to pick up the latest version. You could try a repush to see if it picks up the latest dmd and in case it doesn't you could change the examples to not be `SPEC_RUNNABLE_EXAMPLE_COMPILE`

Reply:
Done